### ISSUE TITLE
Small questions about parameter settings

### ISSUE THREAD

**Issue by zmuzai** [PRIS-CV/Mutual-Channel-Loss#17]
`def supervisor(x,targets,height,cnum):
        mask = Mask(x.size(0), cnum)
        branch = x
        branch = branch.reshape(branch.size(0),branch.size(1), branch.size(2) * branch.size(3))
        branch = F.softmax(branch,2)
        branch = branch.reshape(branch.size(0),branch.size(1), x.size(2), x.size(2))
        branch = my_MaxPool2d(kernel_size=(1,cnum), stride=(1,cnum))(branch)  
        branch = branch.reshape(branch.size(0),branch.size(1), branch.size(2) * branch.size(3))
        loss_2 = 1.0 - 1.0*torch.mean(torch.sum(branch,2))/cnum # set margin = 3.0

        branch_1 = x * mask 

        branch_1 = my_MaxPool2d(kernel_size=(1,cnum), stride=(1,cnum))(branch_1)  
        branch_1 = nn.AvgPool2d(kernel_size=(height,height))(branch_1)
        branch_1 = branch_1.view(branch_1.size(0), -1)

        loss_1 = criterion(branch_1, targets)
        
        return [loss_1, loss_2] `
what dose cnum mean？ cmun is  mean channels number of the feature or class numbers of dataset?
if I want to input data like [batch, channel, W , H] = [?,   32,  1,  40], How should I set this parameter cmun。
`def Mask(nb_batch, channels):

    foo = [1] * 2 + [0] *  1
    bar = []
    for i in range(200):
        random.shuffle(foo)
        bar += foo
    bar = [bar for i in range(nb_batch)]
    bar = np.array(bar).astype("float32")
    bar = bar.reshape(nb_batch,200*channels,1,1)
    bar = torch.from_numpy(bar)
    bar = bar.cuda()
    bar = Variable(bar)
    return bar`
Can you explain the meaning of "200" of the above code ?
Looking forward to your answer！
Thank you very much！

**Comment by dongliangchang** [comment]
about cnum : please refer to https://github.com/PRIS-CV/Mutual-Channel-Loss/issues/2 for details.
about 200 :  which equal to the total number of categories.

**Comment by zmuzai** [comment]
get, thank you
